### ISSUE TITLE
Installed npm resolving to incorrect path

### ISSUE THREAD

**Issue by ArkadyDR** [volta-cli/volta#776]
Installing a custom npm version as default isn't working for me (nor is pinning it). 

Running volta 0.8.2 on Windows 10 Build 17763.

```
λ volta install npm@6.13.4
success: installed and set npm@6.13.4 as default
```

```
λ volta list all
⚡⚡️ User toolchain:

    Node runtimes:
        v12.15.0 (default)

    Package managers:
        npm:
            v6.13.4 (default)

    Packages:
```

```
λ npm -v
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module 'C:\Users\me\AppData\Local\Volta\tools\image\npm\6.13.4\bin\node_modules\npm\bin\npm-cli.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module 'C:\Users\me\AppData\Local\Volta\tools\image\npm\6.13.4\bin\node_modules\npm\bin\npm-cli.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

The path it's trying to use doesn't match what is actually within my Volta AppData dir:

```
λ ls 'C:\Users\me\AppData\Local\Volta\tools\image\npm\6.13.4\bin'
node-gyp-bin  npm  npm-cli.js  npm.cmd  npx  npx-cli.js  npx.cmd```
```

also
```
λ volta uninstall npm@6.13.4
warning:  No package 'npm@6.13.4' found to uninstall

λ volta uninstall npm
error: Uninstalling npm is not supported yet.
```

If I just install node and not npm, npm resolves correctly to the version bundled with node (which happens to also be 6.13.4 in this case).

**Comment by charlespierce** [comment]
@ArkadyDR Thanks for the report! We actually have to do some customizing of the scripts that `npm` uses to launch. It looks like something got crossed when that was implemented and it was updating the wrong file. Opened #777 to fix that and write the correct file.

**Comment by charlespierce** [comment]
@ArkadyDR Just released Volta 0.8.3, which should include this fix. After installing it, you likely need to delete everything under `%LocalAppData%\Volta\tools\image\npm\` (since anything there was created with the bugged version). Make sure to leave the `npm` folder itself, but anything underneath it can be deleted and will be regenerated (correctly this time) as needed.

**Comment by ArkadyDR** [comment]
@charlespierce thanks for the super quick turnaround on that. I've tested the new version and it's now working as expected.
